docker/orthanc/orthanc.json

```diff
@@ -14,22 +14,27 @@
     "ConnectionRetryInterval": 5,
     "IndexConnectionsCount": 2
   },
-  "Dicom-DateTruncation": {
-    "dateformat": ["YYYY","04","10"]
-  },
-  "Dicom-Filter": {
-    "blacklist": [
-      "0010"
-    ],
-    "whitelist": [
-      "0010,0030"
-    ]
+  "DataAnon": {
+    "HardlinksUseHashBins" : true,
+    "Hardlinks": {
+      "/by-study-date/": "0008,0020",
+      "/by-pid/": "0010,0020",
+      "/by-dob/": "0010,0030"
+    },
+    "DateTruncation": {
+      "default": "YYYYMM01",
+      "0010,0030": "YYY0MM01"
+    },
+    "Filter": {
+      "blacklist": [
+        "0010"
+      ],
+      "whitelist": [
+        "0010,0030"
+      ]
+    }
   },
```

Comment:
This would have been nice to move into its own config file. Then we could have detected it's presence and created a default file if missing.